web/src/routes/wohnen/components/OfferDetail.js

```diff
@@ -3,13 +3,12 @@
 import React from 'react'
 import { WohnenOfferModel, WohnenFormData, type AccommodationType } from 'api-client'
 import styled from 'styled-components'
-import type { TFunction } from 'react-i18next'
-import { withTranslation } from 'react-i18next'
+import { withTranslation, TFunction } from 'react-i18next'
 import Caption from '../../../modules/common/components/Caption'
 
 type PropsType = {|
   offer: WohnenOfferModel,
-  t: TFunction
+  t: typeof TFunction
```

Comment:
Isn't TFunction already a type? Seems weird that you have to do typeof.

I think we declared this type manually in the flow-typed directory.

Reply:
I think there are no types yet for react-i18next. https://issues.tuerantuer.org/browse/IGAPP-203

Reply:
Added info to remove " t: typeof TFunction" after adding types to Task IGAPP-203